### PR TITLE
Attempt:Backend get request to retrive comments

### PR DESCRIPTION
**Pull request by dolivas644**
GOAL: Retrieve comment section for specific anime page

### CODE REVIEW

File: server/routes/comments.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import db from "../db/Db-connection.js";
+import express from "express";
+const router = express.Router();
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+router.get('/', async (req, res) => {
+  
+      try {
+        const itemList = await db.any(`SELECT * FROM comment WHERE anime_id=$1`, [anime_id]);
```
Comment (dolivas644) [inline server/routes/comments.js:10]: Tested query on postico and it is working!

File: server/routes/comments.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import db from "../db/Db-connection.js";
+import express from "express";
+const router = express.Router();
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+router.get('/', async (req, res) => {
+
```
Comment (dolivas644) [inline server/routes/comments.js]: Should i define anime_id here?

**Comment by dolivas644** [comment]
Attempted request on postman 
URL: http://localhost8080/comments?anime_id=322
RETURNS:
{}

File: server/routes/comments.js (hunk 1 above)
Comment (cdm-ium) [reply server/routes/comments.js:10]: Based on what you are saying here: https://github.com/dolivas644/AniTracker/pull/23#issuecomment-1319324359

The request is working, but not returning any data. 

1. What do you expect the api to return? What data do you have in the db that should come back?
2. Where is animeId coming from? I don't see you parsing the query params at all so I think you need to do that somewhere from the `req` object?
3. What value for `animeId` is coming in here?


It seems like the problem is either `anime_id` isn't right and so the query isn't filtering what you expect or something is off with your query and even though the `anime_id` is what you expect, it isn't working in the query how you expect it to.

**Comment by dolivas644** [comment]
Issue: anime_id is not defined 
FIXED

File: server/routes/comments.js (hunk 1 above)
Comment (cdm-ium) [reply server/routes/comments.js:10]: I'm actually confused how this code would run at all as it looks like `anime_id` isn't defined so shouldn't this throw an error?

**Pull request merged**